report.c

```diff
@@ -238,6 +238,8 @@ void report_grbl_settings() {
   printPgmString(PSTR(" (magazine gap enabled, bool)"));
   printPgmString(PSTR("\r\n$42=")); print_uint8_base10(settings.use_load_cell);
   printPgmString(PSTR(" (use load cell, bool)"));
+  printPgmString(PSTR("\r\n$43=1"));
```

Comment:
Hmm, Ok.  Seems a bit subtle, but this is a smaller change than re-writing the motion components to be a bit more flexible.  Can you just leave a comment with those instructions?